src/applications/vaos/tests/new-express-care-request/info-page.unit.spec.jsx

```diff
@@ -74,12 +84,127 @@ describe('VAOS integration: Express Care info page', () => {
+      .clone()
+      .subtract('2', 'minutes')
+      .tz('America/Denver');
+    const endTime = today
+      .clone()
+      .add('1', 'minutes')
+      .tz('America/Denver');
+    const requestCriteria = getExpressCareRequestCriteriaMock('983', [
+      {
+        day: today
+          .clone()
+          .tz('America/Denver')
+          .format('dddd')
+          .toUpperCase(),
+        canSchedule: true,
+        startTime: startTime.format('HH:mm'),
+        endTime: endTime.format('HH:mm'),
+      },
+    ]);
+    mockRequestEligibilityCriteria(['983'], requestCriteria);
+    mockRequestLimit({
+      facilityId: '983',
+      requestLimit: 1,
+      numberOfRequests: 1,
+    });
+    const router = {
+      push: sinon.spy(),
+      replace: sinon.spy(),
+    };
+    const store = createTestStore(initialState);
+    const screen = renderInReduxProvider(
+      <NewExpressCareRequestLayout router={router} location={location}>
+        <ExpressCareInfoPage router={router} />
+      </NewExpressCareRequestLayout>,
+      {
+        store,
+      },
+    );
+
+    expect(await screen.findByText(/How Express Care Works/i)).to.exist;
+    fireEvent.click(await screen.getByText(/^Continue/));
+    await waitFor(() => expect(router.push.called).to.be.false);
+    expect(
+      await screen.findByText(
+        /You’ve reached the limit for Express Care requests/i,
+      ),
+    ).to.exist;
+  });
+
+  it('should display error if request limit fetch fails', async () => {
+    const today = moment();
+    const startTime = today
+      .clone()
+      .subtract('2', 'minutes')
+      .tz('America/Denver');
+    const endTime = today
+      .clone()
+      .add('1', 'minutes')
+      .tz('America/Denver');
+    const requestCriteria = getExpressCareRequestCriteriaMock('983', [
+      {
+        day: today
+          .clone()
+          .tz('America/Denver')
+          .format('dddd')
+          .toUpperCase(),
+        canSchedule: true,
+        startTime: startTime.format('HH:mm'),
+        endTime: endTime.format('HH:mm'),
+      },
+    ]);
+    mockRequestEligibilityCriteria(['983'], requestCriteria);
+    mockRequestLimit({
+      facilityId: '983',
+      requestLimit: 1,
+      numberOfRequests: 1,
+    });
+    setFetchJSONFailure(
+      global.fetch.withArgs(
+        `${
+          environment.API_URL
+        }/vaos/v0/facilities/983/limits?type_of_care_id=CR1`,
+      ),
+      { errors: [] },
+    );
+    const router = {
+      push: sinon.spy(),
+      replace: sinon.spy(),
+    };
+    const store = createTestStore(initialState);
+    const screen = renderInReduxProvider(
+      <NewExpressCareRequestLayout router={router} location={location}>
+        <ExpressCareInfoPage router={router} />
+      </NewExpressCareRequestLayout>,
+      {
+        store,
+      },
+    );
+
+    expect(await screen.findByText(/How Express Care Works/i)).to.exist;
+    fireEvent.click(await screen.getByText(/^Continue/));
```

Comment:
The `getBy*` functions are synchronous, so `await`ing them doesn't have any effect (but also doesn't break because non-promises are just returned, I believe).